README.md

```diff
@@ -73,12 +73,11 @@ dataset loading script as explained
 
 Trapper supports the common NLP metrics through 
 [jury](https://github.com/obss/jury). Jury is an NLP library dedicated to provide 
```

Comment:
Can we replace this:
`We implement the trapper components by trying to use the available components of the transformers library as much as we can. `
with
`While implementing the components of trapper, we try to reuse the classes from the transformers library as much as we can.`